src/app/pages/HomePage/index.styles.tsx

```diff
@@ -0,0 +1,24 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  main: ({ spacings, mq }: Theme) =>
+    css({
+      margin: `0 ${spacings.FULL}rem`,
+      [mq.GROUP_2_MIN_WIDTH]: {
+        margin: `0 ${spacings.DOUBLE}rem`,
+      },
+    }),
+  inner: css({
+    maxWidth: '63rem',
+    margin: '0 auto',
+  }),
+  inline: ({ mq }: Theme) =>
+    css({
+      [mq.GROUP_4_MIN_WIDTH]: {
+        alignItems: 'center',
+        display: 'flex',
+      },
+    }),
```

Comment:
We might be able to delete this now - don't think it's used anywhere

Reply:
It was used in the TopicPage for the topic title / description, which we don't need on our homepage
https://github.com/bbc/simorgh/blob/09492650a1ff81e4234a1d162314eac14c8c38bb/src/app/pages/TopicPage/TopicPage.jsx#L82

Reply:
Removed, thanks! 😄